src/Umbraco.Cms.Api.Management/Controllers/Package/EmptyPackageController.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Management.ViewModels.Package;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Packaging;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Package;
+
+public class EmptyPackageController : PackageControllerBase
+{
+    private readonly IUmbracoMapper _umbracoMapper;
+
+    public EmptyPackageController(IUmbracoMapper umbracoMapper)
+    {
+        _umbracoMapper = umbracoMapper;
+    }
+
+    /// <summary>
+    ///     Gets an empty package to use as a scaffold when creating a new package.
+    /// </summary>
+    /// <returns>The empty package.</returns>
+    [HttpGet("empty")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PackageDefinitionViewModel), StatusCodes.Status200OK)]
+    public async Task<ActionResult<PackageDefinitionViewModel>> Empty()
```

Comment:
I don't think we need this endpoint anymore, since the frontend now knows about our models through the open API schema